README.md

```diff
@@ -0,0 +1,23 @@
+# Purr Data packaged for Flatpak
```

Comment:
I'm generally against READMEs which duplicate official documentation.

Reply:
I don't spend much time working with flatpak-builder, so the 'How to build' is useful for me to remember how to use it! I can remove it if there's a problem. Or if there's an online quick-reference for flatpak-builder somewhere I can just link to that.